Sources/Swift/Protocol/SentryViewControllerBreadcrumbTracking.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+///
+/// Use this protocol to customize the name used in the automatic
+/// UIViewController performance tracker.
+///
+@objc
+public protocol SentryViewControllerBreadcrumbTracking: NSObjectProtocol {
```

Comment:
`h`: We use `SwiftDescriptor getObjectClassName` in multiple locations in our code base for getting the UIViewControllerName; see https://github.com/search?q=repo%3Agetsentry%2Fsentry-cocoa%20SwiftDescriptor%20getObjectClassName&type=code

I think we should use `SentryViewControllerBreadcrumbTracking` in all of these scenarios for consistency. Therefore, we should rename this to `SentryUIViewControllerName` and update the code comment above accordingly.

Reply:
Ok i will change, i was thinking about some generic protocol in case in future you would like to add some additional parameters which might be send with breadcrumb. If we name it UIViewControllerName and in future we would like to add additional parameters which might, propably we would have to rename it?